modules:
  robot:
    JointStatePerceptionModule: robot_joints
  humans:
    OptitrackPerceptionModule: optitrack
  objects:
    ArTrackPerceptionModule: ar_track
    Pr2GripperPerceptionModule:
      - pr2_left_gripper
      - pr2_right_gripper
    StaticObjectsPerceptionModule: static

robot_joints:
  min_period : 0.09
  robot_name: pr2

pr2_left_gripper:
  side: 0 # = left
  pressure_threshold: 400
  distance_threshold: 0.03

pr2_right_gripper:
  side: 1 # = right
  distance_threshold: 0.03

optitrack:
  name: human_0
  offset_x: 6.4868
  offset_y: 2.8506

static:
  file: package://overworld/config/env/adream.yaml